13 refact mention comments (#53)
comp129/comp55: docs/labs/13-Refact.md

**<u>The amount of repetition that you have in your code will impact
your final project grade.</u>**

That said, it is equally important to ensure that any code you write is easily understandable.
If you find that anything you write is not immediately clear,

> take the time to add comments that explain the purpose and function of the code in question.

Reviewer: ```suggestion
take the time to add comments that explain the code's purpose/intent.
```
